case1.php

```diff
@@ -11,60 +11,149 @@
 	</head>
 	<body>
 		<main>
-			<form method="post">
+
+		<div class="maincontainer">	
+	<div class="header1"><h3>&nbsp; &nbsp; &nbsp; Vul hier uw verbruik in&nbsp; &nbsp; &nbsp; </h3>
+	</div>
+
+	<div class="header2"><h3>&nbsp; &nbsp; &nbsp; Hieronder staat uw kosten overzicht &nbsp; &nbsp; &nbsp; </h3>
+	</div>
+	</div  >
+			<form class="energyInput" method="post">
+					<div class="period">       
+						<select required id="month" name="month">
+						<option value = "">Select month</option>
+						<?php
+							
+							for ($month = 1; $month <= 12; $month++) {
+								$monthName = date("F", mktime(0, null, null, $month, 1));
+								echo '<option value='.$month.'>'.$monthName.'</option>';
+								
+							}
+							
+						
+    					?>
+						</select>
+					</div>
+					<div class="energyUsage">
+						<input required type="number" name="period">
+						
+					</div>
+					<div class="submit">
+						<button type="submit">Bereken</button>
+					</div>	
+			</form><br>				
+	<div class="energycost">
+	<?php	
+
+						
+		function calc(){
+			
+			
+			
+			$month = "";
+			$energyCost = "";
+			$energyTax = 0;
+			if (isset($_POST['month'])) { $month = getMonth($_POST['month']); }
+			if (isset($_POST['period'])) { $energyCost = energyUsageCalc($_POST['period']);}
+			if (isset($_POST['period'])) { $energyTax = tax($_POST['period']); }
+
+			echo " Uw verbruik is " . "in" . " "  . $month . "is" ." " . $energyCost . "<br>";
```

Comment:
echo " Uw verbruik " . "in" . " "  . $month . "is  €" ." " . $energyCost . "<br>";